src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,9 +1,270 @@
+                        />
+                      </div>
+                    );
+                  })}
+              </div>
+              <div
+                className="slideList"
+                style={{
+                  transform: `translate3d(
+                ${imageNum}px, 0px, 0px`,
+                }}
+              >
+                {productInfoList.images &&
+                  productInfoList.images.map(image => {
+                    return (
+                      <span className="slideContent" key="image.id">
+                        <ProductPhoto imageUrl={image.image_url} />
+                      </span>
+                    );
+                  })}
+              </div>
+            </div>
+            <div className="productText">
+              <div className="productName">
+                {productInfoList && productInfoList.name}
+              </div>
+              <div className="productPrice">
+                <div className="productColor">
+                  {productInfoList.colors && productInfoList.colors[0].name}
+                  <span className="productColorDivision">•</span>
+                  {productInfoList.colors && productInfoList.colors[0].name}
+                </div>
+                <div>
+                  {productInfoList.price &&
+                    productInfoList.price
+                      .toString()
+                      .replace(/\B(?=(\d{3})+(?!\d))/g, ',')}
+                  원
+                </div>
+              </div>
+              <div
+                className="bonus"
+                onClick={() => {
+                  this.setState({ isBonus: false });
+                }}
+              >
+                카드 혜택보기
+              </div>
+              <div className="colorNumber">
+                {productInfoList.colors && productInfoList.colors.length} 개의
+                색상
+              </div>
+              <div className="colorBox">
+                {productInfoList.colors &&
+                  productInfoList.colors.map(color => {
+                    return (
+                      <div className="outCircle" key={color.id}>
+                        <div
+                          className={`color${color} circle`}
+                          style={{
+                            backgroundColor: color.name,
+                          }}
+                        ></div>
+                      </div>
+                    );
+                  })}
+              </div>
+              <div
+                className="productSize"
+                onClick={() => {
+                  this.setState({ isSize: false });
+                }}
+              >
+                <button>
+                  SIZE &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{size}
+                  <i className="fas fa-chevron-right"></i>
+                </button>
+              </div>
+              <div
+                className={size ? 'productPickOff' : 'productPickOn'}
```

Comment:
```suggestion
                className={`productPick ${size ? 'off' : 'on'}`}
```
css 쪽 확인하니까 중복되는 스타일이 많은거 같아요 별도의 셀렉터로 분리하면 중복을 해소할 수 있을거 같습니다

Reply:
더 중복제거가 되는군요....감사합니다🌈